tests/test_scheduler.py

```diff
@@ -102,6 +101,23 @@ def fixture_frontend():
     return FP if HAVE_FP else OFP
 
 
+def graphviz_present():
```

Comment:
If this needs to be available for more than one test, we can put it in conftest.py

Reply:
Fixed and added a bit more explanation